metadata-ingestion/src/datahub/ingestion/source/usage/bigquery_usage.py

```diff
@@ -911,16 +906,17 @@ def _get_bigquery_log_entries_via_gcp_logging(
 
         i: int = 0
         entry: Union[AuditLogEntry, BigQueryAuditMetadata]
-        for i, entry in enumerate(
-            heapq.merge(
-                *list_entry_generators_across_clients,
-                key=self._get_entry_timestamp,
-            )
-        ):
-            if i == 0:
-                logger.info("Starting log load from GCP Logging")
-            self.report.total_log_entries += 1
-            yield entry
+        for list_of_entries in list_entry_generators_across_clients:
```

Comment:
Please put the `heap.merge` back. Otherwise, the pre-aggregation results will not be correct across clients.